src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -79,7 +77,7 @@ internal class FunctionInstanceLogger : IAsyncCollector<FunctionInstanceLogEntry
                         // This exception will cause the function to not get executed.
                         throw new InvalidOperationException($"Missing function.json for '{shortName}'.");
                     }
-                    state = new FunctionInstanceMonitor(descr.Metadata, _metrics, item.FunctionInstanceId, descr.Invoker.FunctionLogger);
+                    state = new FunctionInstanceMonitor(descr.Metadata, _metrics, item.FunctionInstanceId);
```

Comment:
why don't we need the invoker specific logger anymore?

Reply:
That 'Logger' was a special Logger that Mike had created to combine together the `TraceWriter` and `ILogger`. It was called a 'Logger' but wasn't actually an `ILogger`. Since we don't have to send logs between multiple logging infrastructures anymore, we don't need it.